attribute/parser.go

```diff
@@ -1,45 +1,44 @@
 package attribute
 
 import (
-	"log"
+	"fmt"
 	"time"
 
 	"github.com/getyoti/yoti-go-sdk/v2/yotiprotoattr"
 )
 
-func parseValue(contentType yotiprotoattr.ContentType, byteValue []byte) (result interface{}) {
+func parseValue(contentType yotiprotoattr.ContentType, byteValue []byte) (interface{}, error) {
 	switch contentType {
 	case yotiprotoattr.ContentType_DATE:
 		parsedTime, err := time.Parse("2006-01-02", string(byteValue))
+
 		if err == nil {
```

Comment:
Should we return in here? Or change to `!== nil` and return the error?

Reply:
Good spot, not sure why it's like this!